web/src/components/userpage/event-review-card-list.js

```diff
@@ -0,0 +1,101 @@
+import EventReviewCard from './event-review-card';
+import {gql} from 'apollo-boost';
+import {Grid, Typography} from '@material-ui/core';
+import PropTypes from 'prop-types';
+import {Query} from 'react-apollo';
+import React, {Component} from 'react';
+import {SAMPLE_EVENTS} from './sample-data';
+
+export const USER_EVENTS_QUERY = gql`
+query getUserEvents($info: String!) {
+  userEvents(info: $info) {
+    id
+    title
+    thumbnailUrl
+    schedule {
+      startDateTime
+      endDateTime
+      address
+    }
+    ... on EventBookClub {
+      bookTitle
+      bookAuthor
+    }
+    reviews {
+      title
+      content
+    }
+  }
+}
+`
+
+// TODO(mskwon1): Implement GraphQL Query statements.
+class EventReviewCardList extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  // Render EventReviewCard components list.
+  renderEventReviewCards(events) {
+    // Map event objects to EventReviewCard component.
+    events = events.map((event) => {
+      const {
+        id, 
+        title: eventTitle, 
+        bookTitle, 
+        bookAuthor, 
+        thumbnailUrl: bookImage, 
+        schedule: schedules, 
+        reviews: review
+      } = event;
+      const {currentTab} = this.props;
+      
+      return (
+        <EventReviewCard 
+          key={id}
+          id={id} 
+          eventTitle={eventTitle} 
+          bookTitle={bookTitle}
+          bookAuthor={bookAuthor}
+          bookImage={bookImage}
+          schedules={schedules} 
+          review={review} 
+          upcoming={currentTab === 'upcoming' ? true : false}
+        />
+      );
+    });
+
+    // Render the components.
+    return events;
+  }
+
+  sendUserEventsQuery() {
+    return (
+      <Query 
+        query={USER_EVENTS_QUERY}
```

Comment:
this can go to above line

Reply:
```suggestion
      <Query query={USER_EVENTS_QUERY}
```
like this? just to make sure